packages/material-ui/src/TablePagination/TablePaginationActions.d.ts

```diff
@@ -5,6 +5,10 @@ export interface TablePaginationActionsProps extends React.HTMLAttributes<HTMLDi
   backIconButtonProps?: Partial<IconButtonProps>;
   count: number;
   nextIconButtonProps?: Partial<IconButtonProps>;
+  /**
+   * @deprecated Use onPageChange instead.
+   */
+  onChangePage: (event: React.MouseEvent<HTMLButtonElement> | null, page: number) => void;
   onPageChange: (event: React.MouseEvent<HTMLButtonElement> | null, page: number) => void;
```

Comment:
```suggestion
  onChangePage?: (event: React.MouseEvent<HTMLButtonElement> | null, page: number) => void;
  onPageChange?: (event: React.MouseEvent<HTMLButtonElement> | null, page: number) => void;
```

Shouldn't these be optional?